bundle/edu.gemini.p1monitor/src/main/resources/edu/gemini/p1monitor/config/conf.production-2015A-kr.xml

```diff
@@ -1,5 +1,5 @@
 <conf>
-    <smtp>smtp.cl.gemini.edu</smtp>
+    <smtp>sbfsmtprelay-lv1.cl.gemini.edu</smtp>
 
```

Comment:
smtp.cl.gemini.edu is the name we need to keep actually - it's an alias. It will point to sbfsmtprelay-lv1.cl.gemini.edu when the move to Google is done. So we shouldn't change this at all, except if we want to test the Google relay server from this application before the actual migration. I'm not sure it's necessary. It will be useful to confirm the machine this service runs is authorized to send emails to sbfsmtprelay-lv1.cl.gemini.edu - but that can be done using telnet. 

So, this PR should :-1: